src/main/java/com/ldtteam/blockout/Pane.java

```diff
@@ -249,14 +249,6 @@ public void disable()
         setEnabled(false);
     }
 
-    /**
-     * Is pane visible and enabled.
-     */
-    public boolean isOn()
```

Comment:
hm is this used nowhere in minecolonies either?

Reply:
I added it before year approximately so I doubt anyone used it. Like you shouldn't need to use it anytime